mempool.go

```diff
@@ -396,6 +400,13 @@ func (mp *txMemPool) addTransaction(utxoView *blockchain.UtxoViewpoint, tx *btcu
 		},
 		StartingPriority: calcPriority(tx.MsgTx(), utxoView, height),
 	}
+
+	// If the fee estimator is not null, record this tx for fee estimation.
+	if mp.cfg.FeeEstimator != nil {
```

Comment:
Please put this code after `atomic.StoreInt64(&mp.lastUpdated, time.Now().Unix())`.  Putting it here makes the above comment no longer apply to the block and also it's an optional thing just like the `mp.cfg.AddrIndex != nil` below so it fits better there.

Also, for the comment, please just say `Record this tx for fee estimation if enabled.` so it matches the style of the `mp.cfg.AddrIndex != nil` that will be right next to it after it's moved.